Plugin~/WebRTCPlugin/DummyAudioDevice.cpp

```diff
@@ -30,10 +30,12 @@ namespace webrtc
         if (!initialized_)
             return 0;
 
+        initialized_ = false;
```

Comment:
@karasusan i've recently seen editor crashing after stopping playmode, I've reapplied this patch into my fork as I haven't seen it happen since :(

Reply:
though if it's ok, i'll probably have a small fix PR coming up and could land it as a part of it